libraries/ms-helper/src/thermistor.c

```diff
@@ -2,9 +2,8 @@
 #include <stdint.h>
 #include "log.h"
 
-// resistance in milliohms
-// src: https://www.murata.com/en-global/products/productdata/8796836626462/NTHCG83.txt
-static const uint32_t resistance_lookup[] = {
+// expected resistance in milliohms for a given temperature in celsius
+static const uint32_t s_resistance_lookup[] = {
```

Comment:
Why did you remove the source?

Reply:
The model number is in `thermistor.h`, I thought you said that it was unnecessary to include a link to the datasheet?

Reply:
Unnecessary in the header. It's good practice to show where magic values come from.